spellbook/models/balances/ethereum/balances_ethereum_erc20_day.sql

```diff
@@ -0,0 +1,49 @@
+{{ config(
+        alias='erc20_day'
+        )
+}}
+
+with
+    days as (
+        select
+            explode(
+                sequence(
+                    to_date('2015-01-01'), date_trunc('day', now()), interval 1 day
+                )
+            ) as day
+    )
+
+, daily_balances as
+ (SELECT
```

Comment:
are we using a linter for this?

Reply:
I think that was just me trying out a linter (one shot to see if the jinja syntax was taken into account), but the idea is to include something like SQL fluff directly into github actions soon
https://dune.height.app/T-10880